tests/ui/new_without_default.stderr

```diff
@@ -31,8 +31,6 @@ LL | /     pub fn new() -> LtKo<'c> {
 LL | |         unimplemented!()
 LL | |     }
    | |_____^
-   |
-   = note: `-D clippy::new-without-default` implied by `-D warnings`
```

Comment:
Any clue to why this went away?

Reply:
This note only appears when the lint is first emitted. Before the merge, every error message above was about `new-without-default-derive`. Look at the first error message, you'll now find this note there.

So this is expected 👍